test/e2e/Progress.test.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+/* eslint-disable
+  no-undef
+*/
+const fs = require('fs');
+const path = require('path');
+const testServer = require('../helpers/test-server');
+const reloadConfig = require('../fixtures/reload-config/webpack.config');
+const runBrowser = require('../helpers/run-browser');
+
+describe('client progress', () => {
+  describe('using hot', () => {
+    const cssFilePath = path.resolve(__dirname, '../temp/main.css');
+    beforeAll((done) => {
+      fs.writeFileSync(
+        cssFilePath,
+        'body { background-color: rgb(0, 0, 255); }'
+      );
+      const options = {
+        port: 9000,
+        host: '0.0.0.0',
+        inline: true,
+        hot: true,
+        progress: true,
+        watchOptions: {
+          poll: 500,
+        },
+      };
+      testServer.startAwaitingCompilation(reloadConfig, options, done);
+    });
+
+    afterAll((done) => {
+      testServer.close(done);
```

Comment:
fs.unlinkSync(cssFilePath);

Reply:
need to add `fs.unlinkSync(cssFilePath);`

Reply:
done